pkg/kubelet/kuberuntime/logs/logs.go

```diff
@@ -454,15 +450,15 @@ func waitLogs(ctx context.Context, id string, w *fsnotify.Watcher, runtimeServic
 		case e := <-w.Events:
 			switch e.Op {
 			case fsnotify.Write:
-				return true, false, nil
-			case fsnotify.Create:
 				fallthrough
 			case fsnotify.Rename:
 				fallthrough
 			case fsnotify.Remove:
 				fallthrough
 			case fsnotify.Chmod:
-				return true, true, nil
+				return true, false, nil
+			case fsnotify.Create:
+				return true, filepath.Base(e.Name) == logName, nil
```

Comment:
fallthrough looks a bit strange here. Isn't it the same as
```
case fsnotify.Write, fsnotify.Rename, snotify.Chmod:
        return true, false, nil
```
?

Reply:
Just followed the previous code style.  I have changed.